media-libs/ctl/files/ctl-1.5.2-cmake-fixes.patch

```diff
@@ -0,0 +1,217 @@
+diff -purN a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
The other patches have absolutely great patch metadata (backport from upstream are the best), for this one it is still missing

Reply:
That's a patch I made that is being submitted upstream. I'll pull the patch out of git tomorrow. On my phone at the moment as BT is being weird so no internet for me.